media/css/firefox/challenge-the-default/_animations.scss

```diff
@@ -0,0 +1,121 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$campaign-red: #ff6a75;
+$campaign-pink: #ffbdc5;
+$campaign-mint: #d1ffee;
+$campaign-green: #2ac3a2;
+
+.js {
+    @media (prefers-reduced-motion: no-preference) {
+
+        @keyframes pop-in {
+            0% {
+                opacity: 0;
+                transform: scale(0.3);
+            }
+
+            35% {
+                opacity: 1;
+                transform: scale(0.3);
+            }
+
+            50% {
+                transform: scale(0.7);
+            }
+
+            65% {
+                transform: scale(0.7);
+            }
+
+            80% {
+                transform: scale(1.1);
+            }
+
+            90% {
+                transform: scale(1.1);
+            }
+
+            100% {
+                opacity: 1;
+                transform: scale(1);
+            }
+        }
+
+        @keyframes close {
+            0% {
+                opacity: 1;
+                transform: scale(1);
+            }
+
+            50% {
+                opacity: 0.5;
+                  transform: scale(0.7);
+            }
+
+            100% {
+                opacity: 0;
+                 transform: scale(0.3);
+            }
+        }
```

Comment:
**Suggestion:** I also played around with making this animation smoother. I added more changes on the percent scale, give it a shot:

```
@keyframes close {
            0% {
                opacity: 1;
                transform: scale(1);
            }

            25% {
                opacity: 0.75;
                transform: scale(0.9);
            }

            50% {
                opacity: 0.5;
                transform: scale(0.7);
            }

            65% {
                transform: scale(0.5)
            }

            80% {
                opacity: 0.3;
                transform: scale(0.3);
            }

            100% {
                opacity: 0;
                transform: scale(0);
            }
        }
```